source/use-nunjucks-macros/index.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: Use Nunjucks macros
+weight: 45
+---
+
+# Use Nunjucks macros
```

Comment:
I wonder if this should just be called 'Use Nunjucks' as it covers two things that work in different ways:

- Extending (and configuring) the Nunjucks page template (which is not a macro)
- Using the component macros

Introducing them as two distinct things then breaking them into their own sections might help with explaining the fact that e.g. configuring the page template works differently?

Reply:
Good point, and I'm going to rejig the intro to this page slightly based on this and @mgladdish's second comment above.